analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -899,7 +899,8 @@ def parse_options(compilation_db_entry,
                   keep_gcc_include_fixed=False,
                   keep_gcc_intrin=False,
                   get_clangsa_version_func=None,
-                  env=None):
+                  env=None,
+                  analyzer_clang=None):
```

Comment:
Maybe `analyzer_clang_version` or simply `analyzer_version` would be a better name for this.